test/python/tf_unittest_runner.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+import sys
+import argparse
+import os
+
+#Import ngraph in builds without ngraph embedded
```

Comment:
Why is this step necessary? Currently the Python tests won't work with nGraph loaded separately - so we should not `import ngraph`. Please consider removing this block.